condor_watch_q.py

```diff
@@ -998,36 +1015,46 @@ def strip_ansi(string):
     return ANSI_ESCAPE_RE.sub("", string)
 
 
+PROGRESS_BAR_CHARS_AND_COLORS = {
+    JobStatus.COMPLETED: (Color.GREEN, "#"),
+    JobStatus.RUNNING: (Color.CYAN, "="),
+    JobStatus.IDLE: (Color.BRIGHT_YELLOW, "-"),
+    JobStatus.HELD: (Color.RED, "!"),
+    JobStatus.SUSPENDED: (Color.RED, "!"),
+    JobStatus.REMOVED: (Color.RED, "!"),
+}
+
+PROGRESS_BAR_ORDER = (
+    JobStatus.COMPLETED,
+    JobStatus.RUNNING,
+    JobStatus.IDLE,
+    JobStatus.HELD,
+    JobStatus.SUSPENDED,
+    JobStatus.REMOVED,
+)
+
+
+def make_row_progress_bar(totals, width=None, color=True):
+    if width < 10:
+        print("TOO NARROW")
+    width -= 2
+    num_total = float(totals[TOTAL])
+    # switch to totals.get() to fix
```

Comment:
This function doesn't appear to do anything - looks like its dead code from a partial solution.